src/dmd/tokens.d

```diff
@@ -24,539 +24,292 @@ import dmd.utf;
 
 enum TOK : int
 {
-    TOKreserved,
+    reserved,
 
     // Other
-    TOKlparen,
-    TOKrparen,
-    TOKlbracket,
-    TOKrbracket,
-    TOKlcurly,
-    TOKrcurly,
-    TOKcolon,
-    TOKneg,
-    TOKsemicolon,
-    TOKdotdotdot,
-    TOKeof,
-    TOKcast,
-    TOKnull,
-    TOKassert,
-    TOKtrue,
-    TOKfalse,
-    TOKarray,
-    TOKcall,
-    TOKaddress,
-    TOKtype,
-    TOKthrow,
-    TOKnew,
-    TOKdelete,
-    TOKstar,
-    TOKsymoff,
-    TOKvar,
-    TOKdotvar,
-    TOKdotid,
-    TOKdotti,
-    TOKdottype,
-    TOKslice,
-    TOKarraylength,
-    TOKversion,
-    TOKmodule,
-    TOKdollar,
-    TOKtemplate,
-    TOKdottd,
-    TOKdeclaration,
-    TOKtypeof,
-    TOKpragma,
-    TOKdsymbol,
-    TOKtypeid,
-    TOKuadd,
-    TOKremove,
-    TOKnewanonclass,
-    TOKcomment,
-    TOKarrayliteral,
-    TOKassocarrayliteral,
-    TOKstructliteral,
-    TOKclassreference,
-    TOKthrownexception,
-    TOKdelegateptr,
-    TOKdelegatefuncptr,
+    leftParentheses,
+    rightParentheses,
+    leftBracket,
+    rightBracket,
+    leftCurly,
+    rightCurly,
+    colon,
+    negate,
+    semicolon,
+    dotDotDot,
+    endOfFile,
+    cast_,
+    null_,
+    assert_,
+    true_,
+    false_,
+    array,
+    call,
+    address,
+    type,
+    throw_,
+    new_,
+    delete_,
+    star,
+    symbolOffset,
+    variable,
+    dotVariable,
+    dotIdentifier,
+    dotTemplateInstance,
+    dotType,
+    slice,
+    arrayLength,
+    version_,
+    module_,
+    dollar,
+    template_,
+    dotTemplateDeclaration,
+    declaration,
+    typeof_,
+    pragma_,
+    dSymbol,
```

Comment:
Is there a reason this can't be called just `symbol`? Is it to match the `Dsymbol` class?

Reply:
> Is it to match the Dsymbol class?

Yes, that was my intention.  I don't have any good reason for it not to be `symbol`.